Added configuration section in restricted editing feature guide (#18)
ckeditor/ckeditor5-restricted-editing: docs/features/restricted-editing.md

{@snippet features/restricted-editing}

## Configuration

> It is possible to configure which features should be available in restricted mode. For instance, following configuration will only allow for bolding, typing and text deletion.

Reviewer: ```suggestion
It is possible to configure which features should be available in the restricted mode. For instance, the following configuration will only allow typing, deleting, and bolding text.
```

```js
import RestrictedEditingMode from '@ckeditor/ckeditor5-restricted-editing/src/restrictededitingmode';
import Bold from '@ckeditor/ckeditor5-basic-styles/src/bold';

ClassicEditor
	.create( document.querySelector( '#editor' ), {
		plugins: [ Bold, RestrictedEditingMode, ... ],
		toolbar: [ 'bold', '|', 'restrictedEditing', ... ],
		restrictedEditing: {
			allowedCommands: [ 'bold' ]
		}
	} )
	.then( ... )
	.catch( ... );
```

> Note, that typing and text deletion is inherently allowed. For more information check {@link module:restricted-editing/restrictededitingmode~RestrictedEditingModeConfig} documentation.

Reviewer: ```suggestion
**Note**: Typing and deleting text is always possible in restricted editing regions. For more information check {@link module:restricted-editing/restrictededitingmode~RestrictedEditingModeConfig} documentation.
```
